sdk/sdk-api/src/main/java/mil/emp3/json/geoJson/GeoJsonExporter.java

```diff
@@ -466,9 +471,14 @@ private void appendFeatureList(List<IFeature> featureList, StringBuffer buffer)
     }
 
     private void getAllFeatures(IOverlay overlay, List<IFeature> featureList) {
-        for (IOverlay ovl : overlay.getOverlays()) {
-            getAllFeatures(ovl, featureList);  // recursive
-            featureList.addAll(ovl.getFeatures());
+         List<IOverlay> overlays = overlay.getOverlays();
+        if (overlays != null && !overlays.isEmpty()) {
+            for (IOverlay ovl : overlay.getOverlays()) {
+                getAllFeatures(ovl, featureList);  // recursive
+                featureList.addAll(ovl.getFeatures());
+            }
+        } else {
+            featureList.addAll(overlay.getFeatures());
```

Comment:
Does it enter this block if overlays is == null?

Reply:
yes

Reply:
Ahh duh, I read overlay.getFeatures as overlays and figured a npe would occur.